syntax = "proto3";

package protocal;

message TranslateRequest {
    string msg = 1;
};

message TranslateResponce {
    string msg = 1;
};

message AddRequest {
    int32 num1 = 1;
    int32 num2 = 2;
};

message AddResponce {
    int32 result = 1;
};